docs/handbook/image-file-formats.rst

```diff
@@ -1124,6 +1124,11 @@ The :py:meth:`~PIL.Image.Image.save` method supports the following options:
 **method**
     Quality/speed trade-off (0=fast, 6=slower-better). Defaults to 4.
 
+**exact**
```

Comment:
These are keyword arguments, shall we document them in alphabetical order for ease of reference?

Reply:
It currently reads,
```
lossless -> quality -> method -> exact -> icc_profile -> exif
```
Is it preferable to read,
```
exact -> exif -> icc_profile -> lossless -> method -> quality
```
?
If we put `exact` before `exif` it breaks the `icc_profile, exif` connection (as the common parameters across different formats.) 
if we put `exact` after `exif` it breaks the connection between the other 3 encoding parameters, `lossless`, `quality`, `method`. I think the sequence `lossless -> quality -> method -> exact` orders the priority of encoding parameters nicely.
Anyway, if you could let me know what you had in mind, I'd be happy to apply the change. thanks.

Reply:
Ah, let's not worry about it and leave it as you have now :)

Thanks!